src/symbolDocumenter.ts

```diff
@@ -20,7 +20,7 @@ const labelDefinitionRegex = /^[\s]*((([a-zA-Z_][a-zA-Z_0-9]*)?\.)?[a-zA-Z_][a-z
 const defineExpressionRegex = /^[\s]*(?:def[\s]*)?([a-zA-Z_][a-zA-Z_0-9]*)[\s]+(equ|equs|set|=)[\s]+.*$/i
 const instructionRegex = new RegExp(`^(${syntaxInfo.instructions.join("|")})\\b`, "i");
 const keywordRegex = new RegExp(`^(${syntaxInfo.preprocessorKeywords.join("|")})\\b`, "i");
-const macroDefinitionRegex = /^macro[\s]+([a-zA-Z_][a-zA-Z_0-9]*).*$/i
+const macroDefinitionRegex = /^\s*macro[\s]+([A-Z_]\w*).*$/i
```

Comment:
Similar to my last comment, adding support for indented macros makes a lot of sense.

I'm a bit confused about the identifier matcher later in the regex, though. The `\w` is awesome, and there's other regexes where it would make sense to make that substitution. But replacing `[a-zA-Z_]` with `[A-Z_]` seems odd to me. Aren't macros allowed to begin with a lowercase letter?

Reply:
They are, which is what the `i` flag at the end of the expression is for. I just thought It would be better to simplify the expression, since I was changing it anyway

Reply:
Oh yes! You're totally right. Good catch. 👍